tests/test_testclient.py

```diff
@@ -45,7 +45,7 @@ def test_use_testclient_in_endpoint():
     assert response.json() == {"mock": "example"}
 
 
-def testclient_as_contextmanager():
```

Comment:
Thanks! Let's go for `test_use_testclient_as_contextmanager`

Reply:
Amended.